### PR TITLE
@feature/backend review model

### PR DESCRIPTION
**Pull request by alexh205**
- Backend Review Model Created
- Backend Review Creation Route Created
- Backend Get All reviews by username  route created
- Backend Get All reviews by bookId route created
- Removed the 'serverTest.js' module
- Updated the '.env.example'
- Added an index.js module to group all backend routes to a single module for importing
- General formatting Throughout.

### CODE REVIEW

File: backend/.env.example [hunk 1]
```diff
@@ -1,2 +1,5 @@
 PORT=5000
-DB_URI='mongodb://localhost:27017/your-database-name'
\ No newline at end of file
+DB_URI='mongodb://localhost:27017/your-database-name'
+BREVO_USERID='your-brevo-user-id'
+REVO_PASS='your-brevo-password'
```
Comment (bank1e) [inline backend/.env.example]: Thank you for adding this to .env.sample!
nits Typo found: 
x  REVO_PASS
o BREVO_PASS

**Comment by bank1e** [comment]
Looks good to me except for the typo in .env.example. Please check the single comment in that file.
Thank you for your work on the backend review model!

Here are summary of testing this PR at my end:

**1. Backend Get All reviews by bookId route created:**
image
👆️ x GET to pull out two reviews for this book. (Tested book ID: _6648a4f00e87e541e8e8bcfb_)

**2. Backend Get All reviews by username route created:**
image
👆️ x GET to pull out one review that this user submitted: (Tested user name: _user12343454_)
image
FYI: User name: _user12343454_ = User ID: _6648b6451395a75b2f724789_

**3. Backend Review Creation Route Created:**
image
👆️ x Negative test scenario (1): If there is no body text, error message shows up.

image
👆️ x Successful post request with review (title, body, rating, userId, bookId).

image
👆️ x Negative test scenario (2): If the user already submitted review for the book, error message shows up.
image
FYI: In database, there is already an existing review against the book ID: _6648b90fd47d451741bcd4b1_

image
image
👆️ x Negative test scenario (3): If book ID/user ID do not match in the record, errro message shows up.

Thank you for house keeping the below existing codes and formatting:

4. Removed the 'serverTest.js' module
5. Updated the '.env.example'
6. Added an index.js module to group all backend routes to a single module for importing
7. General formatting Throughout.

File: backend/.env.example (hunk 1 above)
Comment (alexh205) [reply backend/.env.example]: Thank you for catching the typo. I have just made the correction

**Review by bank1e — APPROVED** [review]
Perfect!

**Pull request merged**